guides/introduction/community.md

```diff
@@ -7,6 +7,8 @@ There are a number of places to connect with community members at all experience
   * We're on Freenode IRC in the [\#elixir-lang](http://webchat.freenode.net/?channels=elixir-lang) channel.
 
   * [Request an invitation](https://elixir-slackin.herokuapp.com/) and join the #phoenix channel on [Slack](https://elixir-lang.slack.com).
+  
+  * Feel free to join and checkout the #phoenix channel on [Discord](https://discord.gg/elixir).
```

Comment:
Someone correct me if I'm wrong, but I think it should be "check out" (as two words) if you intend to use it as a verb.

Reply:
You are correct.

Reply:
Good catch, thanks